LocalWiki.php

```diff
@@ -27,6 +27,35 @@ function onBeforePageDisplay( OutputPage $out ) {
 			$out->addMeta( 'og:image:width', '1200' );
 		}
 
+		break;
+	case 'gratisdatawiki':
+		$wgHooks['OutputPageParserOutput'][] = 'onOutputPageParserOutput';
+		$wgHooks['BeforePageDisplay'][] = 'onBeforePageDisplay';
+		
+		function onOutputPageParserOutput( OutputPage $out, ParserOutput $parserOutput ) {
+			$placeholders = $parserOutput->getExtensionData( 'wikibase-view-chunks' );
+			if ( $placeholders !== null ) {
+				$out->setProperty( 'wikibase-view-chunks', $placeholders );
+			}
+			$termsListItems = $parserOutput->getExtensionData( 'wikibase-terms-list-items' );
+			if ( $termsListItems !== null ) {
+				$out->setProperty( 'wikibase-terms-list-items', $termsListItems );
+			}
+			$meta = $parserOutput->getExtensionData( 'wikibase-meta-tags' );
+			$out->setProperty( 'wikibase-meta-tags', $meta );
+			$alternateLinks = $parserOutput->getExtensionData( 'wikibase-alternate-links' );
+			if ( $alternateLinks !== null ) {
+				foreach ( $alternateLinks as $link ) {
+					$out->addLink( $link );
+				}
+			}
+		}
+				
+		function onBeforePageDisplay( OutputPage $out ) {
+			$out->addMeta( 'og:image:width', '1200' );
+			$out->addMeta( 'revisit-after', '1 days' );
```

Comment:
This isn't used by any major browser anymore.

Reply:
> This isn't used by any major browser anymore.

It should go out then, I got the inspiration from metawiki's own, but since it's going out, then why not

Reply:
I have created #4613 for cleaning meta up

Reply:
Yeah, I saw